resources/ini.ebnf

```diff
@@ -0,0 +1,21 @@
+ini = body section*.
+
+section = header (eol body)?.
+header = "[" wsp name wsp "]" wsp comment?.
+name = #"[\w ]+(?<! )"
+
+body = (kv? wsp comment? eol)* (kv wsp comment?)?
+
+kv = key wsp "=" wsp val.
+key = #"[\w ]*(?<! )"
+val = #"[\w ]*(?<! )"
+
+comment = #"#.*".
+wsp = "(\t| )*".
+eol = "(\r|\n|\r\n)".
+
+blank-line = wsp comment? eol.
```

Comment:
I’m not actually familiar with this file format, just sort of muddling through it based on intuition, but it looks like maybe this isn’t used? If so could we / should we / maybe drop it?

Reply:
You're right, this is a holdover from a previous version. I just coalesced some of the whitespace definitions, making this one moot.